modules/morph-section/main/index.coffee

```diff
@@ -63,7 +63,7 @@ class InlineMorphSection extends MorphSection
         morphSection.detector = undefined
         morphSection.hide()
     @on 'hide', 'hx.morph-section', (data) ->
-      exitEditMode.call(morphSection, data.toggle, data.content)
+      exitEditMode?.call(morphSection, data.toggle, data.content)
```

Comment:
So you don't have to pass it through as an empty function if you don't have an exit edit mode function